src/MensaWidget/MWApiWrapperTest.class.st

```diff
@@ -65,34 +65,46 @@ MWApiWrapperTest class >> validMensaId [
 
 {
 	#category : #tests,
-	#'squeak_changestamp' : 'Louis Christ 6/4/2024 17:22'
+	#'squeak_changestamp' : 'Max 6/18/2024 18:34'
 }
 MWApiWrapperTest >> testApiCallWithInvalidUrl [
 
     | responseWrapper errorOccurred |
 
     errorOccurred := false.
-    [ responseWrapper := MWApiWrapper getMealsFrom: self class invalidMensaId on: self class invalidDate.
-    ] on: NetworkError do: [ :ex |
-        errorOccurred := true ].
+    [responseWrapper := MWApiWrapper getMealsFrom: self class invalidMensaId on: self class 	invalidDate using: WebClient] on: NetworkError do: [:ex | errorOccurred := true].
```

Comment:
maybe do some linebreaks or formatting here 

```
[responseWrapper := MWApiWrapper getMealsFrom: self class invalidMensaId on: 
   self class invalidDate using: 
   WebClient] on: 
NetworkError do: [:ex | errorOccurred := true].
```

maybe like this as an example